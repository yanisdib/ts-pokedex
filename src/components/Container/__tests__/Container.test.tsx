import { render } from '@testing-library/react';

import Container from '../Container';


describe('Container component', () => {
    test('should render without crashing', () => {
        const { container } = render(<Container />);

        expect(container).toMatchSnapshot();
    });
});