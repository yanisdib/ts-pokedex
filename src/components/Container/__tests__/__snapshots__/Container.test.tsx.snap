// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component should render without crashing 1`] = `
<div>
  <div
    class="sc-bczRLJ gZiCIE"
  />
</div>
`;
